cmake-modules/global_compile_options.cmake

```diff
@@ -4,7 +4,8 @@ if(MSVC)
   endif()
 
   #https://stackoverflow.com/questions/37527946/warning-unreferenced-inline-function-has-been-removed
-  add_compile_options(/W4 ${WARNINGS_AS_ERRORS_FLAG} /wd5031 /wd4668 /wd4820 /wd4255 /wd4710 /analyze)
+  # C28020 and C28204 are introduced by nlohmann/json
+  add_compile_options(/W4 ${WARNINGS_AS_ERRORS_FLAG} /wd5031 /wd4668 /wd4820 /wd4255 /wd4710 /wd28020 /wd28204 /analyze)
```

Comment:
I don't feel it's right to disable warnings for the whole project just because of a warning from a header. Can we reduce the scope?

Reply:
Actually we cannot, the reason is that 28204 is being reported from the system header and it needs to be added to where the first time this header is included, which can be changed over time.
This is, after all, a temporary solution before JSON parser is supported in Azure Core so I would not be concerned about it.

Reply:
At least we can move it to compile options of azure storage library?